indico/modules/categories/blueprint.py

```diff
@@ -46,6 +49,15 @@ def _redirect_event_creation(category_id, event_type):
 _bp.add_url_rule('/manage/protection', 'manage_protection', RHManageCategoryProtection, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/settings', 'manage_settings', RHManageCategorySettings, methods=('POST', 'GET'))
 
+# Role management
+_bp.add_url_rule('/manage/roles', 'manage_roles', RHCategoryRoles, methods=('POST', 'GET'))
+_bp.add_url_rule('/manage/roles/create', 'add_role', RHAddCategoryRole, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/roles/<int:role_id>/edit', 'edit_role', RHEditCategoryRole, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/roles/<int:role_id>', 'delete_role', RHDeleteCategoryRole, methods=('DELETE',))
```

Comment:
Why not having all the `HTTP` methods handled by the same `RH`?

Reply:
i asked him to just copy from the event roles RHs ;) if we move this to react at some point i'd use that moment to switch to the usual REST-style rh structure